src/Microsoft.Extensions.Logging/Microsoft.Extensions.Logging.csproj

```diff
@@ -17,6 +17,8 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Configuration" Version="$(AspNetCoreVersion)" />
     <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="$(AspNetCoreVersion)" />
+    <PackageReference Include="Microsoft.Extensions.Options" Version="$(AspNetCoreVersion)" />
+    <PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="$(AspNetCoreVersion)" />
```

Comment:
"M.E.O.ConfigurationExtensions" brings in some of the above as transitive dependencies. I think it's more declarative to have those here as well - but not sure how you handle these things?!?